packages/mainsail/source/confirmed-transaction.dto.ts

```diff
@@ -59,7 +59,9 @@ export class ConfirmedTransactionData extends DTO.AbstractConfirmedTransactionDa
 	}
 
 	public override fee(): BigNumber {
-		return this.bigNumberService.make(this.data.fee);
+		const gasLimit = this.bigNumberService.make(this.data.gasLimit);
+		const gasPrice = this.bigNumberService.make(this.data.gasPrice);
+		return gasLimit.times(gasPrice);
```

Comment:
@ItsANameToo @goga-m Please correct me if the calculation is incorrect

Reply:
technically correct, but it depends on the UI to show it properly in terms of value